src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -179,7 +181,9 @@
         List<Document> documents = new ArrayList<>();
         documents.add(digitalGrantDocument);
         documents.add(coverSheet);
-
+        if(letterOfGrantIssuedState != null){
```

Comment:
The ordering is wrong according to the JIRA

Reply:
This is not actual order, actual order of print has been re-arranged in bulk print service. Originally we were adding grant then cover.